Description: move away from deprecated platform functions
 The linux_distribution function from the platform has been deprecated in
 Python 3.5. `distro` is the natural sucessor, however it is a new dependency.
Author: Antonio Terceiro <terceiro@debian.org>

--- a/pastebinit
+++ b/pastebinit
@@ -38,8 +38,8 @@ defaultPB = "pastebin.com"
 
 # Now try to override it with a distributor pastebin
 try:
-    import platform
-    release = platform.linux_distribution()[0].lower()
+    import distro
+    release = distro.linux_distribution(full_distribution_name=False)[0].lower()
     if release == 'debian':
         defaultPB = "paste.debian.net"
     elif release == 'fedora':
